cstar/base/log.py

```diff
@@ -0,0 +1,41 @@
+import logging
+
+
+DEFAULT_LOG_FORMAT = "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+
+
+def get_logger(
+    name: str, level: int = logging.INFO, format: str | None = None
+) -> logging.Logger:
+    """Get a logger instance with the specified name.
+
+    Parameters
+    ----------
+    name : str
+        The name of the logger.
+
+    Returns
+    -------
+    logging.Logger
+        A logger instance with the specified name.
+    """
+    logger = logging.getLogger(name)
+
+    # Ensure handlers are not duplicated
+    if len(logger.handlers):
+        return logger
+
+    fmt = format or DEFAULT_LOG_FORMAT
+    logging.basicConfig(level=logging.DEBUG, format=fmt)
```

Comment:
```suggestion
    logging.basicConfig(level=level, format=fmt)
```
Looks like the passed in level never gets used? And I guess that is where we can set the default -- do we want debug or info?

Reply:
derp. edit fail. fixed.